### PR TITLE
Adjust milvus interface

### PR DESCRIPTION
**Pull request by junjiejiangjjj**
Signed-off-by: junjiejiangjjj <junjie.jiang@zilliz.com>

### CODE REVIEW

File: towhee/functional/mixins/milvus_mixin.py [hunk 1]
```diff
@@ -19,11 +19,13 @@
 from ..entity import Entity
 
 
-def _milvus_insert(iterable: Iterable, index: Tuple[str], collection, batch_size: int): # pragma: no cover
+def _milvus_insert(iterable: Iterable, index: Tuple[str], host, port,
+                   collection_name, batch_size: int = 100):  # pragma: no cover
```
Comment (reiase) [inline towhee/functional/mixins/milvus_mixin.py:23]: @shiyu22 has implemented a url for milvus db, can we reuse the url here?

**Review by reiase — CHANGES_REQUESTED** [review]

File: towhee/functional/mixins/milvus_mixin.py [hunk 2]
```diff
@@ -65,33 +67,37 @@ def _milvus_insert(iterable: Iterable, index: Tuple[str], collection, batch_size
...
+            port = kws['port']
 
-        if 'collection' in kws:
-            collection = kws['collection']
-        if 'batch' in kws:
-            batch_size = int(kws['batch'])
+        if 'collection_name' in kws:
+            collection_name = kws['collection_name']
+
+        batch_size = 100
```
Comment (reiase) [inline towhee/functional/mixins/milvus_mixin.py:89]: the batch_size should be read from parameters

File: towhee/hub/builtin/operators/milvus_search.py [hunk 3]
```diff
@@ -55,13 +56,12 @@ class milvus_search:
...
      <Entity dict_keys(['path', 'img', 'vec', 'results'])>]
     """
-    def __init__(self, collection, **kwargs):
-        from towhee.utils.milvus_utils import Collection
 
-        if isinstance(collection, str):
-            self.collection = Collection(collection)
-        elif isinstance(collection, Collection):
-            self.collection = collection
+    def __init__(self, collection_name, **kwargs):
```
Comment (reiase) [inline towhee/hub/builtin/operators/milvus_search.py:60]: We should support both pymilvus collections and collection names here, according to the feedback when we review the code examples with the Milvus team.

**Review by reiase — CHANGES_REQUESTED** [review]

File: towhee/functional/mixins/milvus_mixin.py (hunk 1 above)
Comment (shiyu22) [reply towhee/functional/mixins/milvus_mixin.py:23]: Yes, you can refer to it to [parse uri](https://github.com/towhee-io/towhee/blob/main/towhee/connectors/ann_index/milvus.py#L58).

**Pull request closed without merging**